docs/src/pages/en/in-the-wild.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Media Chrome in the wild
+description: Links to other resources for all things Media Chrome and the ideas behind Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+<h2>Using Media Chrome</h2>
+
+Here are some projects and web applications that currently use Media Chrome:
+
+[Mux Player](https://mux.com/player)
+
+Player by Caffeine.tv on [brobible.com](https://brobible.com/sports/article/mike-tyson-made-30-million-biting-evander-holyfield/)
+
+
+
+<h2>Talks, blog posts and other resources</h2>
+
+Here are some resources that relate to the evolving ideas, development, & history of Media Chrome.
+
+[Designing a player for the context and UX you want](https://tmi.mux.com/sessions/designing-a-player-for-the-context-and-ux-you-want) -- Christian Pillsbury, TMI, 2022
```

Comment:
Do we want to link to the author's bio somewhere? Always nice to be able to learn who the person is behind the work.

Reply:
That's a good idea. I'm going to skip that for now, but I think this showcase page is something we will iterate on.

Mostly right now I want to have it so that we have a place to put things as they come up, and accept PRs from outside folks adding their things to it. We should continue to improve it!